lib/transpiler.js

```diff
@@ -28,7 +29,8 @@ module.exports = function (opts) {
   this.stream = function () {
     var stream = replace(/\/\/\s+transpile:(main|mocha)\s*/g, this.header);
     stream.pipe(babel(this.babelOpts))
-          .pipe(renameEsX());
+          .pipe(renameEsX())
+          .pipe(gutil.noop());
```

Comment:
@sebv i already fixed this problem yesterday (i thought). at least it fixed it for me locally. we may not need this noop after I fixed the piping.